File: Java Fundamentals/Basic Sytax Tasks/Task5_Login.java
import java.util.Scanner;
//100/100
public class Task5_Login {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String username = scanner.nextLine();
        String password = "";
        int counter = 0;

        for (int i = username.length() - 1; i >= 0; i--) {

            password += username.charAt(i);
        }

        while (counter < 4) {
            String input = scanner.nextLine();
            if (input.equals(password)) {
                System.out.printf("User %s logged in.", username);
                return;
            } else if (counter < 3) {
                System.out.println("Incorrect password. Try again.");
            }
            counter++;
        }
        if (counter == 4) {
            System.out.printf("User %s blocked!", username);
        }
    }


}

File: C#/C# Fundamentals/Basic Syntax, Conditional Statements & Loops/Exercise Tasks/Task5_Login.cs
using System;

namespace Task5_Login
{
    class Task5_Login
    {
        static void Main(string[] args)
        {
            string username = Console.ReadLine();

            string password = "";
            int counter = 0;


            for (int i = username.Length -1; i >= 0; i--)
            {
                password += username[i];
            }

            
            while (counter < 4)
            {
                string input = Console.ReadLine();
                if (input.Equals(password))
                {
                    Console.WriteLine("User {0} logged in.", username);
                    return;
                }
                else if (counter < 3)
                {
                    Console.WriteLine("Incorrect password. Try again.");
                }
                counter++;
            }
            if (counter == 4)
            {
                Console.WriteLine("User {0} blocked!", username);
            }
        }
        
    }
}
